src/openforms/prefill/tests/test_attributes_generation.py

```diff
@@ -0,0 +1,130 @@
+from textwrap import dedent
+
+from django.test import SimpleTestCase
+
+from requests_mock import Mocker
+
+from ..attributes_generator import (
+    AttributeGeneratorException,
+    OpenApi3AttributesGenerator,
+)
+from ..contrib.haalcentraal.tests.utils import load_binary_mock
+
+
+class TestOpenApi3AttributesGenerator(SimpleTestCase):
+    def test_no_specifications_provided(self):
+        generator = OpenApi3AttributesGenerator()
+
+        with self.assertRaises(
+            AttributeGeneratorException,
+            msg="No URL to the OAS specifications was provided.",
+        ):
+            generator.generate_attributes()
+
+    @Mocker()
+    def test_wrong_schema_provided(self, m_request_oas):
+        m_request_oas.get(
+            "https://personen/api/schema/openapi.yaml?v=3",
+            status_code=200,
+            content=load_binary_mock("personen.yaml"),
+        )
```

Comment:
I also realised that if you use the imported `load_binary_mock`, it uses the path in the `haalcentraal` folder. I added it to some util file

Reply:
I don't understand? If you use `zgw_consumers.test.mock_service_oas_get`, you don't need any `load_binary_mock`. This `load_binary_mock` is already duplicated twice in master, with this PR this would be a third time. In this situation, for downloading the Open API spec, there is already existing tooling in the zgw-consumers package that does exactly that. You then also don't need the symlink, because it's configured in the settings which folders to scan for the file.

And actually, reasoning more about it - can't we just use file paths directly when testing the generator itself, rather than hitting the network and/or dealing with network mocks?

```py
parse("./src/openforms/prefill/contrib/haalcentraal/tests/files/personen.yaml")
```
works fine for me locally.

Even for the management command you can pass in the filepath as URI and things should work as expected :)